### DISCUSSION TITLE
Assets not shown after demo installation

### DISCUSSION THREAD

**Discussion by halmeriarsa** [pimcore/pimcore#11781] [Q&A / Support]
After install pimcore in my mac, i run the front from ../public/.
It seems that something is wrong because any image is loaded :(
![Captura de pantalla 2022-03-28 a las 16 24 05](https://user-images.githubusercontent.com/98109012/160419683-2d0b7531-fc49-4014-a18b-e5d3309bfb49.png)

After some googling, i found this solution (force hard copy of assets)
![Captura de pantalla 2022-03-28 a las 16 24 42](https://user-images.githubusercontent.com/98109012/160419757-d1de6b4f-1995-43f5-adc1-d215c5121b06.png)
But it doesn´t work.

I have not a web/ or assets/ folder in my project.

Could anyone help me, please?

**Comment by halmeriarsa** [comment]
FIXED.

**Reply by dvesh3** [reply]
please explain how you fixed it? Otherwise it doesn't make sense to keep it in the discussions. thanks!
